systemtest/src/main/java/io/strimzi/systemtest/cli/KafkaCmdClient.java

```diff
@@ -17,7 +17,6 @@
 
     private static final int PORT = 9092;
 
-    public KafkaCmdClient() { }
```

Comment:
I think there should be a `private` constructor.

Reply:
Does it make sense to have private Constructor? Afaik this one public empty is default in java, so it's fine if it's not there at all..